### ISSUE TITLE
`eager: true` break the mask

### ISSUE THREAD

**Issue by mrleblanc101** [beholdr/maska#133]
### Describe the bug
Be this mask:
```
telMask: {
    mask: (val) => (val.startsWith('1') ? '1 (###) ###-####' : '(###) ###-####'),
    eager: true,
},
```
Typing `1` should results in `1 (` but results in `1 (1`
Then, if you type a full mask, you can't delete !
![2023-02-03 10 50 49](https://user-images.githubusercontent.com/17392251/216647532-56e9ce3a-b286-4e7d-aad2-915e9d1fcf60.gif)

**Comment by mrleblanc101** [comment]
The mask break when you try to delete any caracter that was added by `eager: true`.
Another exemple where I added `eager: true` so that the space would be automatically added.

Be this mask:
```
postalMask: {
    mask: '@#@ #@#',
    eager: true,
    tokens: {
        '@': { pattern: /[a-zA-Z]/, transform: (chr) => chr.toUpperCase() },
    },
},
```
![2023-02-03 11 06 41](https://user-images.githubusercontent.com/17392251/216650902-0c6adaa3-675b-4c75-910a-8bcf49f1b47e.gif)

**Comment by beholdr** [comment]
Could you please provide online sandbox with this example? I can’t reproduce this case

**Comment by mrleblanc101** [comment]
@beholdr https://codesandbox.io/s/musing-faraday-bwcokq?file=/src/App.vue

**Comment by beholdr** [comment]
Thanks, confirmed in Chrome, but not in FF. Will check.

**Comment by beholdr** [comment]
Should be fixed in v2.1.7
